src/grpc/charts.rs

```diff
@@ -41,16 +42,23 @@ impl chart_server::Chart for ChartService {
         };
 
         let timeframe = Timeframe::from_repr(timeframe).unwrap_or(Timeframe::Unspecified);
-        let result = VoteSummary::get_for_timeframe(timeframe, category, conn!()).await;
 
-        match result {
-            Ok(summaries) if summaries.is_empty() => {
+        let chart = get_chart_cached(category, timeframe).await;
+
+        match chart {
+            Ok(chart) if chart.data.is_empty() => {
                 Err(Status::not_found("Cannot find data for given timeframe."))
             }
 
-            Ok(summaries) => {
-                let chart = Chart::new(timeframe, summaries);
-                let ordered_chart_data = chart.data.into_iter().map(|cd| cd.into()).collect();
+            Ok(chart) => {
+                if chart.was_cached {
+                    info!(
+                        "Using cached chart data for category '{:?}' in timeframe '{:?}'",
+                        category, timeframe
+                    );
+                }
```

Comment:
We said we'd look into the top level task periodically logging cache metrics at a later date but are we wanting to keep this info level logging each time a cached chart is served?

Reply:
Ah sorry, this made it's way back in during the rebase - will remove it for now